### PR TITLE
Simplify project view

### PR DESCRIPTION
**Pull request by rudolfs**
Move everything that is sub-view related into the respective views.

### CODE REVIEW

File: src/views/projects/Patch.svelte [hunk 1]
```diff
@@ -136,7 +142,8 @@
   }));
 
   $: currentRevision =
-    patch.revisions.find(r => r.id === revision) || patch.revisions[0];
+    patch.revisions.find(r => r.id === (revision ?? latestRevision)) ||
+    patch.revisions[0];
```
Comment (geigerzaehler) [inline src/views/projects/Patch.svelte:146]: How about this:
```suggestion
    (revision && patch.revisions.find(r => r.id === revision) || patch.revisions[patch.revisions.length - 1]
```
Then you can get rid of `latestRevision` and the intention becomes much clearer.

File: src/views/projects/Issue/New.svelte [hunk 2]
```diff
@@ -51,9 +49,20 @@
           assignees: utils.stripDidPrefix(assignees),
           tags: tags,
         },
-        session.id,
+        sessionId,
       );
-      dispatch("create", result.id);
+
+      void router.push({
```
Comment (geigerzaehler) [inline src/views/projects/Issue/New.svelte:55]: I’d move this call after the try/catch block. If the call fails, it’s not an authentication error and the issue actually has been created so the error message would confuse the user.

**Review by geigerzaehler — COMMENTED** [review]
Looks great! I have two suggestions

File: src/views/projects/Patch.svelte (hunk 1 above)
Comment (rudolfs) [reply src/views/projects/Patch.svelte]: 069d0a16ce2fc8355c363dbbb80477a365240ffb

**Review by geigerzaehler — APPROVED** [review]

**Pull request merged**